dir.props

```diff
@@ -15,10 +15,12 @@
     <ObjDir Condition="'$(ObjDir)'==''">$(BinDir)obj\</ObjDir>
     <TestWorkingDir Condition="'$(TestWorkingDir)'==''">$(BinDir)tests\</TestWorkingDir>
     <PackagesOutDir Condition="'$(PackagesOutDir)'==''">$(BinDir)packages\</PackagesOutDir>
-    
+
     <!-- Input Directories -->
     <PackagesDir Condition="'$(PackagesDir)'==''">$(ProjectDir)packages\</PackagesDir>
     <ToolsDir Condition="'$(ToolsDir)'==''">$(PackagesDir)Microsoft.DotNet.BuildTools.$(BuildToolsVersion)\lib\</ToolsDir>
+
+    <DirectorySeparator Condition="'$(DirectorySeparator)'==''">$([System.IO.Path]::DirectorySeparatorChar)</DirectorySeparator>
```

Comment:
I was using it in the other file- figured it was good to be set globally so others could do the same thing. That said, it was pointed out that the only issue was the config file. As such I'm going even simpler here.